prisma/schema.prisma

```diff
@@ -232,3 +234,27 @@ model Photo {
   regionId     String?
   eventGroupId String?
 }
+
+model PeerJudgement {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  username  String
+  eventId   String
+  project   Project @relation(fields: [projectId], references: [id])
+  projectId String
+
+  @@unique([username, eventId, projectId])
+}
+
+model Kudos {
```

Comment:
What is a kudo?

Reply:
Unimplemented, but basically that comments system we were talking about.